kola/harness.go

```diff
@@ -169,15 +170,24 @@ func NewFlight(pltfrm string) (flight platform.Flight, err error) {
 		flight, err = packet.NewFlight(&PacketOptions)
 	case "qemu":
 		flight, err = qemu.NewFlight(&QEMUOptions)
+	case "qemu-unpriv":
+		flight, err = unprivqemu.NewFlight(&QEMUOptions)
 	default:
 		err = fmt.Errorf("invalid platform %q", pltfrm)
 	}
 	return
 }
 
-func filterTests(tests map[string]*register.Test, pattern, platform string, version semver.Version) (map[string]*register.Test, error) {
+func filterTests(tests map[string]*register.Test, pattern, pltfrm string, version semver.Version) (map[string]*register.Test, error) {
```

Comment:
Minor: any reason for the variable rename here?

Reply:
I chose to rename it because having a variable named `platform` as well as a `platform` package used in the same file felt like it could lead to confusion; better to just short-name the variable.

Reply:
Gotcha, makes sense!